python/lsst/ts/standardscripts/base_track_target.py

```diff
@@ -347,6 +374,22 @@ async def run_block(self):
                 time_on_target=self.config.track_for,
                 slew_timeout=self.slew_timeout,
             )
+
+        elif self.slew_type == SlewType.PLANET:
+            planet_name = self.config.slew_planet["planet_name"]
+            planet_enum = Planets[planet_name]
+
+            self.log.info(
+                f"Slew and track planet_name={planet_name} rot_value={self.config.rot_value} "
+                f"rot_type={self.config.rot_type} "
```

Comment:
Since this is always `rot_sky`, replace this with `rot_sky={...}` and remove the `rot_type`.